.schema/config.schema.json

```
@@ -957,6 +968,17 @@
           "examples": [
             "{\"subject\":\"{{ .Subject }}\"}"
           ]
+        },
+        "allowed_remote_headers": {
```

I think this should have a different name to more clearly indicate what it is doing. What do you think about:

```suggestion
        "forward_response_headers_to_upstream": {
```